logger/sdkLogger.go

```diff
@@ -0,0 +1,336 @@
+package logger
```

Comment:
This is a new package addition, but the core lib's readme is missing the documentation for the major classes in this namespace. Please add the documentation to the readme file

Reply:
added readme docs